<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="icon_only_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="home_icon1">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../img/iclickart_logo.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="home_btn1">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../img/icons8-home-24.png</normaloff>../img/icons8-home-24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refresh_btn1">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../img/icons8-refresh-16.png</normaloff>../img/icons8-refresh-16.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="search_btn1">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../img/icons8-search-16.png</normaloff>../img/icons8-search-16.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settings_btn1">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../img/icons8-settings-16.png</normaloff>../img/icons8-settings-16.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>584</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_btn1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>36</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../img/icons8-x-16.png</normaloff>../img/icons8-x-16.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="full_menu_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>165</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="home_icon2">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../img/iclickart_logo.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="home_btn2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Home</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refresh_btn2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>새로고침</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="search_btn2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>검색</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settings_btn2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>설정</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>584</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_btn2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>36</height>
          </size>
         </property>
         <property name="text">
          <string>종료</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="content_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>55</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="spread_btn">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../img/icons8-menu-24.png</normaloff>../img/icons8-menu-24.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>903</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="page_widget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="main_page">
          <widget class="QWidget" name="ui_file_list_widget" native="true">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>90</y>
             <width>351</width>
             <height>541</height>
            </rect>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>740</x>
             <y>200</y>
             <width>77</width>
             <height>241</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="add_btn">
              <property name="text">
               <string>추가</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="delete_btn">
              <property name="text">
               <string>삭제</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="select_all_btn">
              <property name="text">
               <string>전체 선택</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="send_data_btn">
              <property name="text">
               <string>전송</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="settings_page">
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>240</y>
             <width>191</width>
             <height>141</height>
            </rect>
           </property>
           <property name="text">
            <string>Settings_after</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="help_page">
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>460</x>
             <y>300</y>
             <width>56</width>
             <height>12</height>
            </rect>
           </property>
           <property name="text">
            <string>Help</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>home_btn1</sender>
   <signal>toggled(bool)</signal>
   <receiver>home_btn2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refresh_btn1</sender>
   <signal>toggled(bool)</signal>
   <receiver>refresh_btn2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_btn1</sender>
   <signal>toggled(bool)</signal>
   <receiver>search_btn2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settings_btn1</sender>
   <signal>toggled(bool)</signal>
   <receiver>settings_btn2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>home_btn2</sender>
   <signal>toggled(bool)</signal>
   <receiver>home_btn1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refresh_btn2</sender>
   <signal>toggled(bool)</signal>
   <receiver>refresh_btn1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_btn2</sender>
   <signal>toggled(bool)</signal>
   <receiver>search_btn1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settings_btn2</sender>
   <signal>toggled(bool)</signal>
   <receiver>settings_btn1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_btn2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>748</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>767</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_btn1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>750</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>770</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spread_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_only_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spread_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>full_menu_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
